docs/fundamentals/executor/executor-api.md

```diff
@@ -366,6 +371,55 @@ the Executor, and `**kwargs` is passed to the internal `Flow()` initialisation c
 For more details on these arguments and the workings of `Flow`, see the {ref}`Flow section <flow-cookbook>`.
 ````
 
+### Run Executor in Kubernetes
+You can generate Kubernetes configuration files for your containerized Executor by using the static `Executor.to_k8s_yaml()` method. This works very similar to {ref}`deploying a Flow in Kubernetes <kubernetes>`, because your Executor is wrapped automatically in a Flow and using the very same deployment techniques.
+
+```python
+from jina import Executor
+
+Executor.to_k8s_yaml(
+    output_base_path='/tmp/config_out_folder',
+    port_expose=8080,
+    uses='jinahub+docker://DummyHubExecutor',
+    executor_type=Executor.StandaloneExecutorType.EXTERNAL,
+)
+```
+```shell
+kubectl apply -R -f /tmp/config_out_folder
+```
+The above example will deploy the `DummyHubExecutor` from Jina Hub into your Kubernetes cluster.
+(external-shared-executor)=
+#### External and shared Executor
+The type of stand-alone Executors can be either external or shared. By default, it will be external. An external Executor is deployd alongside a {ref}`Gatway <architecture-overview>`. A shared Executor has no Gateway. Both types of Executor {ref}`can be used directly in any Flow <external-executor>`.
```

Comment:
```suggestion
The type of stand-alone Executors can be either external or shared. By default, it will be external. An external Executor is deployd alongside a {ref}`Gateway <architecture-overview>`. A shared Executor has no Gateway. Both types of Executor {ref}`can be used directly in any Flow <external-executor>`.
```